product-backend/scripts/db_build.sh

```diff
@@ -10,3 +10,4 @@ sed -e "s/{{USERNAME}}/$username/g" example_data.sql > example_data_exe.sql
 
 mysql -h ${DB_HOST} -P ${DB_PORT} -u ${DB_USERNAME} -p${DB_PASSWORD} < create_db_exe.sql
 mysql -h ${DB_HOST} -P ${DB_PORT} -u ${DB_USERNAME} -p${DB_PASSWORD} ${db_name} < example_data_exe.sql
+
```

Comment:
missing `sed` and execution of User.sql file - check with Jakub

Reply:
Not sure what you mean because ``sad`` are there on lines 8 and 9